symengine/printer.h

```diff
@@ -59,7 +59,7 @@ class Precedence : public BaseVisitor<Precedence> {
             if(x.imaginary_ == 1) {
                 precedence = PrecedenceEnum::Atom;
             } else {
-                precedence = PrecedenceEnum::Mul;
+                precedence = PrecedenceEnum::Add;
```

Comment:
This case is `real` part is `0` and `imaginary` part is `not 1`. Shouldn't it be `Mul`?
/cc @isuruf